### PR TITLE
Setup Development Environment

### PR DESCRIPTION
**Pull request by erozonachi**
### What this PR does?

- Initializes a node project
- Installs required node modules
- Sets up a style guide for the project with eslint and Airbnb-style-guide modules

### CODE REVIEW

File: .eslintrc.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
+module.exports = {
+  env: {
+    commonjs: true,
+    es6: true,
+    node: true,
+  },
+  extends: [
+    'airbnb-base',
```
Comment (samarv) [inline .eslintrc.js]: We don't need this kind of hostility in our life.  in the words of Dan Abormov - extend prettier [config](https://prettier.io/docs/en/integrating-with-linters.html). Also, remember to Add add a prettier config file.

File: package.json [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+    "start": "node index.js",
+    "dev": "NODE_ENV=development npm-run-all server"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/bravolabs/bravo-be.git"
+  },
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
```
Comment (samarv) [inline package.json]: change Licence to MIT

File: package.json [hunk 3]
```diff
@@ -0,0 +1,43 @@
+{
+  "name": "bravo-be",
+  "version": "1.0.0",
+  "description": "Healthy office cultures often acknowledge good work from their coworkers and peers. Award your peers with acknowledgements that act like coins/points in Slack when they do awesome things - and never let the acknowledgement of their good work get lost in the shuffle again.",
+  "main": "index.js",
+  "scripts": {
+    "server": "nodemon index.js",
+    "start": "node index.js",
+    "dev": "NODE_ENV=development npm-run-all server"
```
Comment (samarv) [inline package.json]: why do we have `dev` and a `server` script?

File: package.json (hunk 2 above)
Comment (erozonachi) [reply package.json]: Done.

File: .eslintrc.js (hunk 1 above)
Comment (erozonachi) [reply .eslintrc.js]: Done.

File: .eslintrc.js [hunk 4]
```diff
@@ -0,0 +1,21 @@
+module.exports = {
+  env: {
+    commonjs: true,
+    es6: true,
+    node: true,
+  },
+  extends: [
+    'prettier',
```
Comment (samarv) [inline .eslintrc.js]: Can you ad a prettier [config file](https://github.com/prettier/eslint-plugin-prettier)

**Pull request merged**